homework9/hw9task1/iterators_1/merge_sorted_files.py

```diff
@@ -0,0 +1,47 @@
+"""
+Write a function that merges integer from sorted files and returns an iterator
+file1.txt:
+1
+3
+5
+file2.txt:
+2
+4
+6
+#>>> list(merge_sorted_files(["file1.txt", "file2.txt"]))
+[1, 2, 3, 4, 5, 6]
+"""
+from pathlib import Path
+from typing import Iterator, List, Union
+
+
+def sorting_data(iter1, iter2) -> int:
```

Comment:
Let's change `sorting_data` from a fixed number of iterators (2) to an arbitrary number.

You could start form changing `def sorting_data(iter1, iter2) -> int:` to `def sorting_data(*iters) -> int:`

And when it will work (and pass the tests) update the usage.